static/styles/compose.css

```diff
@@ -336,7 +336,7 @@ input.recipient_box {
 }
 
 #subject.recipient_box {
-    width: calc(20% + 14px);
+    width: calc(20% - -14px);
```

Comment:
Both the nature buggy and fixed makes sense to me in this one. I am posting a gif of the nature here.
![calc-1](https://user-images.githubusercontent.com/17237412/36866832-6e9997d6-1db9-11e8-8c70-9ce8e7673cfb.gif)